code/_onclick/hud/screen_objects/text_objects.dm

```diff
@@ -8,3 +8,130 @@
 	screen_loc = "CENTER-7,CENTER-7"
 	maptext_height = 480
 	maptext_width = 480
+
+#define MAX_TIMERS 4
+
+/atom/movable/screen/text/screen_timer
+	var/maptext_style_left = "<span class='maptext' style=font-size:16pt;text-align:center; align='top'>"
```

Comment:
as said in disc missing screen loc
also missing autodoc

Reply:
should have autodoc now